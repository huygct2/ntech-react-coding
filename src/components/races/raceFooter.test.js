import React from 'react';
import renderer from 'react-test-renderer';
import RaceFooter from './raceFooter';

test('RaceFooter component', () => {
  const component = renderer.create(
    <RaceFooter title="View all markets" />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
