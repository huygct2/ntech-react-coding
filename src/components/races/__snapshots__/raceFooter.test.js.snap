// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RaceFooter component 1`] = `
<div
  className="sports__footer"
>
  <div
    className="sports__all"
  >
    <a
      className="has-text-weight-bold is-uppercase"
    >
      View all markets
    </a>
  </div>
  <span>
    Prices indicative until login & may change before bet is accepted
  </span>
</div>
`;
